sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -72,27 +83,122 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             )
         )
 
-        # This is the role that the lambdas will assume
-        # We'll narrow things down later
+########### LAMBDA FUNCTIONS
+
+        # This is where we install dependencies for the lambda functions
+        # We take advantage of something called "lambda layers"
+        os.system("pip install requests -t ./external/python")
+        os.system("pip install python-jose -t ./external/python")
+        layer = lambda_.LayerVersion(self, f"SDSDependencies-{SDS_ID}",
+                                    code=lambda_.Code.from_asset("./external"),
+                                    description="A layer that contains all dependencies needed for the lambda functions"
+                                )
+
+        # This is the role that the lambdas will all assume
+        # TODO: We'll narrow things down later, right now we're just giving admin access
         lambda_role = iam.Role(self, "Indexer Role", assumed_by=iam.ServicePrincipal("lambda.amazonaws.com"))
         lambda_role.add_managed_policy(iam.ManagedPolicy.from_aws_managed_policy_name("AdministratorAccess"))
```

Comment:
👀 Please don't add Administrator access as a blanket policy!! That is not secure. We should follow the principle of least access.